README.md

```diff
@@ -27,6 +28,7 @@ on:
     - '**.restext'   # INI-based, key value pair file format
     - '**.resx'      # XML-based (resource) translation file format, .NET
     - '**.xliff'     # XML-based translation file format, version 2
+    - '**.json'      # JSON-based, key value pair file format
```

Comment:
Wow, this is great - is there anyway you could update the PR to include updates to the README.md file as well that add .json support?